src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqBurdenEstimateRepository.kt

```diff
@@ -51,6 +51,41 @@ class JooqBurdenEstimateRepository(
     override fun addBurdenEstimateSet(groupId: String, touchstoneId: String, scenarioId: String,
```

Comment:
It's a little hard for me to see clearly here, so I'll just ask: Is it as simple as splitting the logic into two functions, and then just calling both of them from the deprecated method? i.e. There are no actual changes here, really?

Reply:
Very nearly. There are a couple of changes, will flag up with comments.